pkg/services/postgresql/common_bind.go

```diff
@@ -110,6 +110,24 @@ func createCredential(
 		port,
 		databaseName,
 	)
+
+	var jdbcTemplate string
+	if sslRequired {
+		jdbcTemplate = 
+			"jdbc:postgresql://%s:%d/%s?user=%s&password=%s&sslmode=require"
```

Comment:
Does the `sslmode=require` work? I only had tries with `ssl=true`. Sorry I have little experience on it.

Reply:
In my local testing it seems to, and is based on https://www.postgresql.org/docs/current/libpq-ssl.html and is consistent with the pre-existing createCredential calls - https://github.com/Azure/open-service-broker-azure/blob/master/pkg/services/postgresql/common_bind.go#L100

Reply:
Looks good.